hawc/apps/animal/models.py

```diff
@@ -788,19 +789,38 @@ class Endpoint(BaseEndpoint):
     animal_group = models.ForeignKey(
         AnimalGroup, on_delete=models.CASCADE, related_name="endpoints"
     )
+    name_term = models.ForeignKey(
```

Comment:
@rabstejnek moved named_term to Endpoint since we're not really using for vocab yet for other data types; can move later if needed